jobs/build/release/Jenkinsfile

```diff
@@ -101,6 +101,12 @@ node {
                         $class: 'BooleanParameterDefinition',
                         defaultValue: false
                     ],
+                    [
+                        name: 'KEEP_RELEASE_CLOSED',
```

Comment:
We can avoid a double negative here by calling it `OPEN_NEXT_RELEASE` which defaults to `true`.

Reply:
:smile: that's exactly what I had before: https://github.com/openshift/aos-cd-jobs/pull/2307/commits/0413d3f62b4546134a103716cd4a83c7dadf76cf

Reply:
It is bikeshedding, so I am good with whatever you decide. But I am curious for the reasons for preferring `KEEP_RELEASE_CLOSED` over `OPEN_NEXT_RELEASE` :smile:

Reply:
hm ... yesterday I just accepted @sosiouxme 's suggestion without giving it much thought, but i kinda prefer `OPEN_NEXT_RELEASE` (default: true) ... simply because that's how we communicate with each other, by telling "let's open the next release", as opposed to "let's keep this release closed"

Reply:
:facepalm: Only now see the conversation before: https://github.com/openshift/aos-cd-jobs/pull/2307#discussion_r465217045, and agree with the reasoning :+1:

Reply:
hopefully it will all be moot soon (when we stop freezing automation at all and just create advisories when the current release is ready).